tests/suites/test_suite_bignum.function

```diff
@@ -143,6 +143,26 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE */
+void mpi_zero_length_buffer_is_null()
+{
+    mbedtls_mpi X;
+    size_t olen;
+
+    mbedtls_mpi_init(&X);
+
+    /* Simply test that the following functions do not crash when a NULL buffer
+     * pointer and 0 length is passed. We don't care much about the return value. */
```

Comment:
I think we should test that the read calls set `X` to 0 (a plausible failure mode would be that they do nothing, leaving whatever value was previously in `X`). But that's out of scope here, so not a blocker.